src/components/Form/Form.jsx

```diff
@@ -1,20 +1,94 @@
 import React, { useState } from 'react';
 import { useFirebase } from '../../hooks/useFirebase';
+import { useForm } from '../../hooks/useForm';
 
 export const Form = () => {
   const { create } = useFirebase('things');
-  const [thing, setThing] = useState('');
+  const [error, setError] = useState('');
+  const [success, setSuccess] = useState(null);
+
+  const [values, handleInputChange, setValues] = useForm({
+    nameItem: '',
+    selectTime: '',
+    lastDate: null,
+  });
+
   const sendToFB = (e) => {
     e.preventDefault();
-    create({ name: thing });
-    e.target.reset();
+    const itemLength = values.nameItem.length;
+    const selectLength = values.selectTime.length;
+
+    if (itemLength && selectLength) {
+      create({
+        name: values.nameItem,
+        time: values.selectTime,
+        lastDate: null,
+      });
+
+      e.target.reset();
+
+      setValues({
+        nameItem: '',
+        selectTime: '',
+        lastDate: null,
+      });
+      setSuccess('Data was send success');
+      setTimeout(() => {
+        setSuccess(null);
+      }, 2000);
+      return;
+    }
+    setError('Fill in all the blanks');
+    setTimeout(() => {
+      setError(null);
+    }, 2000);
   };
 
   return (
     <div>
-      <form onSubmit={sendToFB}>
-        <input type="text" onChange={(e) => setThing(e.target.value)} />
+      <form onSubmit={sendToFB} className="form-item">
```

Comment:
nice use of a `form`! I believe that because the `form` is around everything, we don't need that outer `div`

Reply:
It has been resolved.